rfcs/2021-07-19-8216-multiple-pipelines.md

```diff
@@ -0,0 +1,258 @@
+# RFC 8216 - 2021-07-19 - Multiple Pipelines
+
+Large Vector users often require complex Vector topologies to facilitate the collection and processing of data from many different upstream sources. This results in large Vector configuration files that are hard to manage across different teams. This makes Vector a bad candidate for large teams that require autonomy to facilitate sane collaboration.
+
+## Context
+
+- [RFC 2064](https://github.com/timberio/vector/blob/master/rfcs/2020-03-17-2064-event-driven-observability.md) Event drive observability.
+- [RFC 8288 (PR)](https://github.com/timberio/vector/pull/8400) Adding the `resources` component.
+
+## Scope
+
+### In the scope
+
+- How and where the pipelines are defined
+- How vector reads the pipelines and what are the limitations of pipelines
+- How do we monitor pipelines components
+
+### Out of scope
+
+- How the pipelines should be synchronized between vector instances
+- Pre/post processing of data - The ability to prepare and normalize data.
+- Connecting pipelines together - The ability to take input from another pipeline.
+- Component reuse - The ability to define boilerplate for reuse across many different pipelines. This will likely align with Datadog’s “pipeline catalogue”.
+- Pipeline quotas - The ability to limit how much data a pipeline can send to a sink.
+- Visibility of the components - The ability to give access to any components within the system.
+
+## Pain
+
+- If an admin wants to split responsibilities between `ops` and `devs` by only allowing the `ops` to deal with `sources/sinks` and the `devs`, it's currently not possible.
```

Comment:
This isn't super clear. The idea is to delegate management of pipelines across a matrix of teams and services.